app/services/avatar_generator.rb

```diff
@@ -0,0 +1,12 @@
+class AvatarGenerator
+  URL = 'https://api.adorable.io/avatars/80/'
```

Comment:
Z tego co rozumiem 80 to w tym kontekście rozmiar obrazka. Jeżeli tak, to warto byłoby go wyekstraktować do osobnej stałej (tak, żeby można było łatwiej zmienić rozmiar obrazka, teraz nie jest to takie oczywiste).

`FILE_SIZE = 80` albo
`IMAGE_SIZE = 80`